src/main/java/com/altenheim/kalender/controller/logicController/WebsiteScraperController.java

```diff
@@ -0,0 +1,27 @@
+package com.altenheim.kalender.controller.logicController;
+
+import java.util.Timer;
+import java.util.TimerTask;
+import com.altenheim.kalender.interfaces.IWebsiteScraperController;
+import com.altenheim.kalender.models.SettingsModel;
+
+public class WebsiteScraperController extends TimerTask implements IWebsiteScraperController
+{
+    private SettingsModel settings;
+
+    public WebsiteScraperController(SettingsModel settings)
+    {
+        this.settings = settings;        
+    }
+
+    public void startScraperTask()
+    {
+        var timer = new Timer();  
```

Comment:
Klasse soweit leer, weil sie vom HWR Scraper Team ja grad implementiert wird, hier nur ein Test für einen Task der als Background Worker (also ohne den Mainthread zu blocken) im festgelegten Intervallen das ausführt, was später unten in der run() Methode steht (ist ein Overwrite, Name muss also so bleiben)